### PR TITLE
Horizontal slider

### PR DESCRIPTION
**Pull request by aneeshsharma0671**
Customized Horizontal slider, and added Background animation for circle

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,17 +1,54 @@
...
     <div className="App" style={{backgroundColor:"#f7f7f7"}}>
-      <Header/>
-      <Swiper navigation pagination onSwiper={(swipe)=>console.log(swipe)} style={{width:"100%", height:"100%"}} >
+          <div
+        class="bg"
+        style={{ borderRadius: "100%" }}
+        bggreen = {bggreen}
+      >
+        <div className="logo-dsc" style={{ backgroundColor: "#f0f0f0" }}>
+          <img src={logo} alt="logo" style={{ width: "100%" }}></img>
```
Comment (therealansh) [inline src/App.js]: i think we can add array here for logos too. which could change whenever there is swipe trigger.

Comment (therealansh) [reply src/App.js]: as in
```js
<img src={LOGOS[swiper.realIndex]} />
```

File: src/styles/BgCircle.css [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    --pos1-left: 90vw;
+    --pos1-top: 0vh;
+    --pos2-left: 0vw;
+    --pos2-top: 0vh;
+    --pos3-left: 0vw;
+    --pos3-top: 100vh;
+    --pos4-left: 90vw;
+    --pos4-top: 100vh;
+  }
+.bg{
```
Comment (therealansh) [inline src/styles/BgCircle.css:17]: There was a PR by @Arjun-Dagar , solves the responsiveness problem with the logo. Do check it out as well. Rn the logo circle will not intersect the bigger circle on medium sized devices.

File: src/App.js [hunk 3]
```diff
@@ -1,17 +1,54 @@
...
+         direction={"horizontal"}
+         slidesPerView={1}
+         spaceBetween={30}
+         mousewheel={true}
+         speed={1000}
+         onSwiper={setSwiper}
+         onSlideChange={(swipe)=>{setSlide(swipe.realIndex); setAnimation();}} 
+        style={{width:"100%", height:"100%"}} 
+      >
+        <SwiperSlide><Home/></SwiperSlide>
```
Comment (therealansh) [inline src/App.js]: Also pull changes from master. The navbar is now functional.

**Review by therealansh — CHANGES_REQUESTED** [review]

**Comment by therealansh** [comment]
This is a really nice PR covering most of the animation. I do request some of the changes tho

File: src/App.js (hunk 3 above)
Comment (aneeshsharma0671) [reply src/App.js]: pulled the changes, navbar is now functional

File: src/styles/BgCircle.css (hunk 2 above)
Comment (aneeshsharma0671) [reply src/styles/BgCircle.css:17]: I have checked that pr, for responsiveness for logo,but it is hard coded. so, we have to adopt a different approach as used in BG circle, I am working on that

**Pull request merged**